### PR TITLE
Release/4.2.4

### PR DESCRIPTION
**Pull request by mobilotest**
- Updated README.md with the latest SDK version
- Updated TROUBLESHOOTING-GUIDE.md with the latest SDK fix info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ The Thunderhead SDK for Android supports Android 4.1+ (API 16) and Android Gradl
 **For _migrating_ from version(s) <= 3.0.0 to version(s) 4.0.0+ of the Thunderhead SDK, please see the [plugin migration guide](ORCHESTRATION-PLUGIN-MIGRATION.md) 
 for details on updating the required Gradle plugins.**
 
+**For _migrating_ from version(s) < 5.0.0 of the Thunderhead SDK to version(s) 5.0.0+, please see the [Java8 Migration guide](JAVA8-MIGRATION-GUIDE.md) for details
```
Comment (andreipop) [inline README.md]: please remove

File: README.md [hunk 2]
```diff
@@ -168,7 +185,10 @@ apply plugin: 'com.thunderhead.android.orchestration-plugin'
 android {
     compileSdkVersion 28
     buildToolsVersion '28.0.0'
-
+    compileOptions {
```
Comment (andreipop) [inline README.md]: needs to be removed

File: README.md [hunk 3]
```diff
@@ -107,6 +120,10 @@ apply plugin: 'com.thunderhead.android.orchestration-plugin'
 android {
     compileSdkVersion 28
     buildToolsVersion '28.0.0'
+    compileOptions {
```
Comment (andreipop) [inline README.md]: needs to be removed

File: README.md [hunk 4]
```diff
@@ -51,7 +54,17 @@ repositories {
 apply plugin: 'com.thunderhead.android.orchestration-plugin'
 ```
 		
-4. Update your `build.gradle` to add codeless identity transfer support.
+4. Add Java8 Support
```
Comment (andreipop) [inline README.md]: needs to be removed

File: README.md [hunk 5]
```diff
@@ -654,6 +674,38 @@ To use the codeless push notifications functionality without using FCM directly,
...
+##### Set a non adaptive fallback.
+
+Android (O)reo, Api 26, shipped with a platform bug relating to Adaptive Icons and Notifications. The bug can be seen [here](https://issuetracker.google.com/issues/68716460). 
+The issue was resolved in Api 27 however it was not back ported to the original Oreo Api 26 platform.  
+
+The Thunderhead SDK will optimize your user's App experience by sending Push Notifications with _your_ application's icon when appropriate. In order to avoid the infinite crash
+loop that the above Android bug causes, the Thunderhead SDK will not show the message if a fallback *NON ADAPTIVE* icon is not set at initialization time on Api 26 devices. 
+Changing your application's icon to a non adaptive icon is not required and the fall back is **only required for Api 26**.
+
+The Thunderhead SDK will warn you at init if the icon has not been set by logging the `14019` error. See [Troubleshooting Guide](TROUBLESHOOTING-GUIDE.md)
```
Comment (andreipop) [inline README.md]: needs to link to the exact section under the [TROUBLESHOOTING-GUIDE ](https://github.com/thunderheadone/one-sdk-android/pull/25/files#diff-98d34e7606162d6e4df92a6a24054a6d)

**Comment by mobilotest** [comment]
Fixed and created new clean PR

**Pull request closed without merging**